pkg/api/types.go

```diff
@@ -1202,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
+	// Fs represents a filesystem-based local volume
+	// +optional
+	Fs *LocalFsVolume
```

Comment:
@jsafrane prefers not to overload the Path field.  It can also make sense to have two separate types if they end up requiring different fields to be specified.

If for block support, we end up adding some access type field, I think it would make sense for that to be in the PV and not the specific volume type.